store/src/test/java/org/apache/rocketmq/store/ConsumeQueueTest.java

```diff
@@ -250,7 +250,11 @@ public void testPutMessagePositionInfo_buildCQRepeatedly() throws Exception {
             for (int i = 0; i < totalMessages; i++) {
                 putMsg(messageStore);
             }
-            Thread.sleep(5);
+
+            // Wait consume queue build finish.
+            while (messageStore.dispatchBehindBytes() > 0) {
```

Comment:
5 seconds is pretty long duration... Best to use awaitability to use finer  granularity.

 Awaitility.with().pollInterval(100, TimeUnit.MILLISECONDS).await().until(() -> {
                return store.dispatchBehindBytes() == 0;
            });